Tests/NIOEmbeddedTests/AsyncTestingChannelTests.swift

```diff
@@ -507,18 +507,23 @@ class AsyncTestingChannelTests: XCTestCase {
     }
 }
 
+<<<<<<< Updated upstream
 fileprivate func XCTAsyncAssertTrue(_ predicate: @autoclosure () async throws -> Bool, file: StaticString = #file, line: UInt = #line) async rethrows {
+=======
+#if compiler(>=5.5.2) && canImport(_Concurrency)
+fileprivate func XCTAsyncAssertTrue(_ predicate: @autoclosure () async throws -> Bool, file: StaticString = #filePath, line: UInt = #line) async rethrows {
+>>>>>>> Stashed changes
```

Comment:
Oops, bad merge, I think you want:

```suggestion
fileprivate func XCTAsyncAssertTrue(_ predicate: @autoclosure () async throws -> Bool, file: StaticString = #filePath, line: UInt = #line) async rethrows {
```